### PR TITLE
New predict page

### LINKED ISSUE #36: Links broken and Requirements/Results unclear
In `pretrained_models` section, we have a broken link. Item 2 in the 'Generating predictions with DeepCell.org' section. The offending link is **the available models and data formatting requirements**

3 other points of concern specific to Mesmer:
1) We do not specify resolution of the data in magnification or um per pixel. This is a key factor in the accuracy of predictions and should be called out specifically (a code snippet to help facilitate people rescaling would be ideal)
2) We should call out the option of submitting a blank channel of data if they only have a nuclear stain/marker or vice versa (a small code snippet would be helpful here as well)
3) We do not specify what the "results" are, so in the case of single-channel they are left to guess which is whole-cell and which is nuc. We should call this out explicitly.

### PR DESCRIPTION
**Pull request by ngreenwald**
This PR updates the docs to reflect the new functionality that has been added to the predict page, as well as integration with DCL viewer. Also closes #36

### CODE REVIEW

**Review by willgraf — APPROVED** [review]

**Comment by willgraf** [comment]
Will this PR be updated with new screenshots as well?

**Review by willgraf — APPROVED** [review]

**Review by MekWarrior — CHANGES_REQUESTED** [review]
As presently constructed, this addresses parts of #36 but does not close it.
We should explain order of the results (which channel is what and what values they should expect) at minimum. It would also be helpful (but not required since its on model card) to call out resolution again in the data format section.

**Comment by ngreenwald** [comment]
It was getting kind of wordy so I tried to split it up into categories, ideally ones that would be general enough that they will also apply to the other models once those are ready to go. Let me know what you think

File: pretrained_models/README.md [hunk 1]
```diff
@@ -23,7 +23,17 @@ Each of the models we host has slightly different requirements for input data. P
...
+
+Image types: The Mesmer model was trained on [TissueNet](https://datasets.deepcell.org/), and thus is optimized for segmenting multiplexed tissue samples. It will **not** perform well on data from other modalties, such as H&E, IHC, cell culture, etc. 
+
+Image resolution: For optimal results, the supplied data should have a physical size of 0.5um per pixel. 
+
+Outputs: The Mesmer model produces two segmentation masks. The first segmentation mask contains the whole-cell predictions, and the second contains the nuclear predictions. Within the masks, each cell is assigned a unique integer value, with the pixels which belong to each cell being given that value. 
+
+Note: The DeepCell.org predict page allows users to specify which channel is the nuclear channel and which is the membrane channel, thus supporting data that is not properly formatted. However, all of the other ways to access the Mesmer model (see [Running pre-trained models in the cloud](#running-pre-trained-models-in-the-cloud) and [Running pre-trained models locally](#running-pre-trained-models-locally)) expect the input data to be properly formatted.
```
Comment (MekWarrior) [inline pretrained_models/README.md:36]: This is great, much cleaner. My only remain suggestion would be to switch the link from 'Running pre-trained models in the cloud' to 'Generating predictions with ImageJ'

File: pretrained_models/README.md [hunk 2]
```diff
@@ -93,15 +103,15 @@ Generating data from the website is quite easy.
...
+2. This will take you to the image upload interface. The default pipeline is `Mesmer`. Before submitting your image, make sure you understand [the available models and data formatting requirements](#formatting-data-for-pre-trained-models). Upload your image by dragging it into the upload box, or by clicking within the box and then browsing to find your image. If you'd like to use our example image, click the 'Download Image' button. 
 
 ![image](resources/DeepCell_website_upload.png)
 
 3. Once your image has been successfully uploaded, click `Submit`, and the server will begin processing your data.  
 
 ![image](resources/DeepCell_website_submit.png)
 
-4. Once complete, you can download the results, and then process additional images.  
+4. Once complete, you can either press Download to view the segmentations locally, or press View to open DeepCell Label to view them interactively.
```
Comment (MekWarrior) [inline pretrained_models/README.md:114]: no comma before 'or'

**Review by MekWarrior — APPROVED** [review]
Great changes. Everything reads much better now. 
I have suggested 2 small changes that I'd like to see but they're not worth blocking the merge over.

**Comment by ngreenwald** [comment]
Great, thanks! Will one of you merge it in?
